src/styles/components/containers/_fold.scss

```diff
@@ -159,6 +159,12 @@
       border-width: 1px 1px 1px 1px;
     }
   }
+
+  .panel {
```

Comment:
a bit gross but needed in order to style nested panels without breaking stuff like the color scale picker